AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/User.java

```diff
@@ -51,5 +53,11 @@ public void setAgentId(Integer agentId) {
 
     public boolean isAdmin(){return admin;}
 
+    public boolean isAgent(){return agent;}
+
+
+
```

Comment:
Careful having too many blank lines, would generally only have 1 here between the getters / setters